packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -172,7 +174,7 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
             // user-specified callback to run in that domain
             try {
               communicator.toSpecBridge(domain, 'run:domain:fn', {
-                data,
+                data: data?.args || undefined,
```

Comment:
```suggestion
                args: options?.args || undefined,
```

I think we should change this here and in the spec bridge so that we consistently call it `args` instead of `data`.

Reply:
updated in b06f21635af698d8d7a006e01ac1d04a699a4b42. Also updated the magic in the communicator to look for `args` instead of `data` to omit user defined values from preprocessing serialization. Looks like `run:domain:fn` is the only event impacted